Sprint-1/3-mandatory-interpret/2-time-format.js

```diff
@@ -12,14 +12,52 @@ console.log(result);
 // For the piece of code above, read the code and then answer the following questions
 
 // a) How many variable declarations are there in this program?
+/* There are 6 variable declarations:
+1.Line 1 : const movieLength.
+2.Line 3 : const remainingSeconds.
+3.Line 4 : const totalMinutes.
+4.Line 6 : const remainingMinutes.
+5.Line 7 : const totalHours.
+6.Line 9 : const result.   
+*/
 
 // b) How many function calls are there?
 
+/* There is one function call:
+Line 10 : console.log().
+*/
+
 // c) Using documentation, explain what the expression movieLength % 60 represents
+
+/* The movieLength shows the length of the movie in seconds, the operand % is remaining (Modulo) returns the remaining of the movieLength divided by 60(to convert sec into minutes) but it only gives the remaining of this operation.
+ */
+
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators
 
 // d) Interpret line 4, what does the expression assigned to totalMinutes mean?
 
+/*
+The expression gives the completed total minutes of the movie, by subtract the movie length in sec from the remaining seconds and divide the result by 60 to convert it to minutes
```

Comment:
You're correct. Just to slightly rearrange the wording, you could say 'subtract the remainingSeconds from the movieLength...'